microsim/opencl/requirements.txt

```diff
@@ -0,0 +1,11 @@
+appdirs==1.4.4

```

Comment:
Is it intentional that we have separate requirements files here?

Reply:
yes I think it's nice to keep this separate, as Alex has made the update which means the conda environment still installs this requirements.txt as well